.github/workflows/ci-unix-static-sanitized.yml

```diff
@@ -11,11 +11,7 @@ jobs:
       fail-fast: false
       matrix:
         os: [ubuntu-latest, macos-latest]
-        sanitizer: [ undefined ] # TODO(yguyon): Add address, memory, thread
-
-    env:
-      CC: clang
-      CXX: clang++
```

Comment:
If I don't delete these three lines, I get segmentation faults in the "ubuntu-latest, address" configuration:

https://github.com/AOMediaCodec/libavif/actions/runs/8351405561/job/22859682384?pr=2068

I don't know why.

Reply:
In my testing today the segmentation faults are gone. Strange.

Reply:
I rebased this pull request and the segmentation faults still did not come back. I don't know what changed that fixed the segmentation faults. I guess it will remain a mystery. :-(